clamav.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	s "strings"
+)
+
+func main() {
+	scanResultsArray := runClamdscan("/tmp/input/*", "/tmp/virus")
+	// Look throgh the clamscan output for files marked OK
+	for _, line := range scanResultsArray {
+		if s.Contains(line, "OK") {
+			// Get the file path for the scanned files
+			filePath := s.Split(line, ":")[0]
```

Comment:
I think we should check whether there actually is an element in this slice just in case ClamAV does something funky to prevent the application from panicking 
```suggestion
			parts := s.Split(line, ":")
			if len(parts) < 1 {
			    log.Printf("failed to parse output line %q: line does not contain expected character ':'", s)
			    continue
			}
			filePath := parts[0]
```